README.md

```diff
@@ -33,37 +33,46 @@ Run locally:
 1. Clone the repo found here: (https://github.com/BeccaHyland/mentor_match_api)
 2. Run `bundle` from the CLI.
 3. Run `rails s`
-4. Using `localhost:3000` as the base of your url in browser, add the endpoint of your choice.  See below.
+4. Using `localhost:3000` as the base of your url in browser, add the endpoint of your choice.  See below:
 
 ## API Endpoints
-This is documentation for unsecured prototype endpoints which will be deprecated shortly. Visit [here](https://github.com/BeccaHyland/mentor_match_api/blob/master/README_endpoints.md) for new endpoint documentation.
+This is documentation for the API endpoints.
 
-All successful database requests return JSON in an easy-to-consume format generated by Netflix's [fast_jsonapi gem](https://github.com/Netflix/fast_jsonapi). To supply the Front End app with data, this project provides the following endpoints:
+All successful database requests return JSON in an easy-to-consume format generated by Netflix's [fast_jsonapi gem](https://github.com/Netflix/fast_jsonapi).
+
+For authorization, all requests must include a parameter of `token`.
+#### The value must be the token received by the FE (in the parameters as `token`) in the first request from the user for that login session.
+##### This ensures that the user already "Logged In" - was authenticated by a Github OAuth login and role-verified by the Back End to set the user's permissions (allowed endpoints)
+
+To supply the Front End app with data, this project provides the following endpoints:
 ## <a name="mentor-endpoints"></a>Mentor Endpoints
 ### 1.  All mentors in the database:
-  GET `/api/v1/mentors` *currently unsecured, authorization coming soon*
-  * returns JSON in the following format:
+  GET `/api/v1/mentors`
+  * returns JSON in the following format (all arrays are shown closed except the first, for easier viewing):
+
+![screen shot 2019-01-22 at 11 24 44 am](https://user-images.githubusercontent.com/38568909/51556800-61d71c00-1e38-11e9-992f-ab828e96064a.png)
 
-  <img width="640" alt="screen shot 2018-12-22 at 2 26 45 pm" src="https://user-images.githubusercontent.com/39714935/50378771-06c2c700-05f7-11e9-945a-3bd7af26f190.png">
+  * Note: This endpoint does not require a token, nor authentication.
```

Comment:
"Note: This is the only endpoint that does not require login or sending a token through the Authorization header."